control-plane/stor-port/src/lib.rs

```diff
@@ -4,7 +4,6 @@
 pub mod transport_api;
 /// Common types for the various resources used by the control-plane internal components.
 pub mod types;
-
```

Comment:
spurious removal?

Reply:
I have removed it while removing the code I added before review comments.